components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -118,7 +118,9 @@ public boolean canHandle(HttpServletRequest request) {
         if (log.isTraceEnabled()) {
             log.trace("Inside OpenIDConnectAuthenticator.canHandle()");
         }
-        if (OIDCAuthenticatorConstants.LOGIN_TYPE.equals(getLoginType(request))) {
+
+        String code = request.getParameter(OIDCAuthenticatorConstants.OAUTH2_GRANT_TYPE_CODE);
+        if (code != null && OIDCAuthenticatorConstants.LOGIN_TYPE.equals(getLoginType(request))) {
```

Comment:
Can you use  StringUtils.isNotBlank(code). WDYT?

Reply:
Resolved with https://github.com/wso2-extensions/identity-outbound-auth-oidc/pull/83/commits/f526303fadccc8122aaf11c8edb5b65a286745e7.